### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by wazam**
for pipenv review

### CODE REVIEW

**Review by noahschechter — APPROVED** [review]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,8 @@ FROM python:3-slim-bullseye
 
 COPY . .
 
-RUN pip install --upgrade pip -r requirements.txt
+RUN pip install -u pipenv
+
+RUN pipenv install
 
 CMD python src/main.py
```
Comment (noahschechter) [inline Dockerfile:9]: `CMD pipenv run python src/main.py`

will run `python src/main.py` within the `pipenv` environment which has all of our necessary libraries installed for the program

Comment (wazam) [reply Dockerfile:9]: is `RUN pipenv install` still required after we install with pip?

Comment (wazam) [reply Dockerfile:9]: also, as `FROM python:3-slim-bullseye` is not the full python library, i will test soon and see if an bump up is required

Comment (noahschechter) [reply Dockerfile:9]: yeah, that will install all of our dependencies in our pipenv

Comment (wazam) [reply Dockerfile:9]: updated for in #6

**Pull request merged**